assets.tf

```diff
@@ -18,32 +18,36 @@ resource "template_dir" "manifests" {
-    serviceaccount_pub      = "${base64encode(tls_private_key.service-account.public_key_pem)}"
-    serviceaccount_key      = "${base64encode(tls_private_key.service-account.private_key_pem)}"
-    etcd_ca_cert            = "${base64encode(tls_self_signed_cert.etcd-ca.cert_pem)}"
-    etcd_client_cert        = "${base64encode(tls_locally_signed_cert.client.cert_pem)}"
-    etcd_client_key         = "${base64encode(tls_private_key.client.private_key_pem)}"
-    aggregation_flags       = "${var.enable_aggregation == "true" ? indent(8, local.aggregation_flags) : ""}"
-    aggregation_ca_cert     = "${var.enable_aggregation == "true" ? base64encode(join(" ", tls_self_signed_cert.aggregation-ca.*.cert_pem)) : ""}"
-    aggregation_client_cert = "${var.enable_aggregation == "true" ? base64encode(join(" ", tls_locally_signed_cert.aggregation-client.*.cert_pem)) : ""}"
-    aggregation_client_key  = "${var.enable_aggregation == "true" ? base64encode(join(" ", tls_private_key.aggregation-client.*.private_key_pem)) : ""}"
+  vars = {
+    hyperkube_image        = var.container_images["hyperkube"]
+    pod_checkpointer_image = var.container_images["pod_checkpointer"]
+    coredns_image          = "${var.container_images["coredns"]}${var.container_arch}"
+    etcd_servers           = join(",", formatlist("https://%s:2379", var.etcd_servers))
+    control_plane_replicas = max(2, length(var.etcd_servers))
+    cloud_provider         = var.cloud_provider
+    pod_cidr               = var.pod_cidr
+    service_cidr           = var.service_cidr
+    cluster_domain_suffix  = var.cluster_domain_suffix
+    cluster_dns_service_ip = cidrhost(var.service_cidr, 10)
+    trusted_certs_dir      = var.trusted_certs_dir
+    ca_cert                = base64encode(tls_self_signed_cert.kube-ca.cert_pem)
+    ca_key                 = base64encode(tls_private_key.kube-ca.private_key_pem)
+    server                 = format("https://%s:%s", var.api_servers[0], var.external_apiserver_port)
+    apiserver_key          = base64encode(tls_private_key.apiserver.private_key_pem)
+    apiserver_cert         = base64encode(tls_locally_signed_cert.apiserver.cert_pem)
+    serviceaccount_pub     = base64encode(tls_private_key.service-account.public_key_pem)
+    serviceaccount_key     = base64encode(tls_private_key.service-account.private_key_pem)
+    etcd_ca_cert           = base64encode(tls_self_signed_cert.etcd-ca.cert_pem)
+    etcd_client_cert       = base64encode(tls_locally_signed_cert.client.cert_pem)
+    etcd_client_key        = base64encode(tls_private_key.client.private_key_pem)
+    aggregation_flags      = var.enable_aggregation == true ? indent(8, local.aggregation_flags) : ""
+    aggregation_ca_cert    = var.enable_aggregation == true ? base64encode(join(" ", tls_self_signed_cert.aggregation-ca.*.cert_pem)) : ""
+    aggregation_client_cert = var.enable_aggregation == true ? base64encode(
```

Comment:
Can we revert the formatting here? I think one-liner is more readable.

Reply:
Also, block indentation is messed up.

Reply:
Done!